pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -58,6 +58,8 @@ class RttExecutor : public rclcpp::Executor
     return rclcpp::ok() && running;
   }
 
+  /// Fill in the output RttestResults message with data from the executor
```

Comment:
@clalancette nit: consider explaining in prose what those `RttestResults` are.

Reply:
Updated in eec9408